pkg/kubectl/cmd/util/helpers_test.go

```diff
@@ -213,72 +214,83 @@ func (f *fileHandler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 	res.Write(f.data)
 }
 
+type checkErrTestCase struct {
+	err          error
+	expectedErr  string
+	expectedCode int
+}
+
 func TestCheckInvalidErr(t *testing.T) {
-	tests := []struct {
-		err      error
-		expected string
-	}{
+	testCheckError(t, []checkErrTestCase{
 		{
 			errors.NewInvalid(api.Kind("Invalid1"), "invalidation", field.ErrorList{field.Invalid(field.NewPath("field"), "single", "details")}),
-			`Error from server: Invalid1 "invalidation" is invalid: field: Invalid value: "single": details`,
```

Comment:
Because the testcase was wrong, testing a code path that could never happen:

`checkErr` calls `handleErr` a number of times. In practice this quits the program. In the test though it does not, i.e. the last `handleErr` wins, giving the `Error from server:` prefix.

I fixed this behavior and adapted the test to reality.